RetroBar/Languages/Tiếng Việt.xaml

```diff
@@ -50,12 +62,14 @@
     <s:String x:Key="maximize">Mở rộng</s:String>
     <s:String x:Key="close">_Đóng</s:String>
 
-    <s:String x:Key="show_hidden">Hiện thị cái biểu tưởng ẩn</s:String>
+    <s:String x:Key="show_hidden">Hiện thị biểu tưởng ẩn</s:String>
     <s:String x:Key="hide">Ẩn</s:String>
 
     <s:String x:Key="set_time">_Thay đổi Ngày/Giờ</s:String>
 
     <s:String x:Key="open_folder">&amp;Mở thư mục</s:String>
 
+    <FlowDirection x:Key="flow_direction">TráiQuaPhải</FlowDirection>
```

Comment:
This value shouldn't be localized, it should be either `RightToLeft` or `LeftToRight`. My understanding is that this translates to LeftToRight, which is the default value. If that is the case, could you please remove it? Thanks!

Reply:
ah oke thanks!